Cartfile.resolved

```diff
@@ -1,2 +1,4 @@
 github "BoltsFramework/Bolts-ObjC" "1.9.1"
+github "BoltsFramework/Bolts-Swift" "1.5.0"
```

Comment:
Do we really need 2 Bolts modules now? I assume they provide the same functionality, and if so, shouldn't the be usable interchangeably in ObjC and Swift code?

Reply:
Got it, then let's keep both for now.